core/aws-core/pom.xml

```diff
@@ -33,6 +33,11 @@
             <artifactId>auth</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>profiles</artifactId>
```

Comment:
Why do we need this dependency?

Reply:
You can enable endpoint discovery through the profile file so I have a provider for it that requires the profiles module.